core/bin/external_node/src/config/mod.rs

```diff
@@ -375,11 +375,35 @@ impl RequiredENConfig {
     }
 }
 
+/// Configuration for Postgres database.
+/// While also mandatory, it historically used different naming scheme for corresponding
+/// environment variables.
+/// Thus it is kept separately for backward compatibility and ease of deserialization.
+#[derive(Debug, Clone, Deserialize, PartialEq)]
+pub struct PostgresConfig {
+    pub database_url: String,
+    pub max_connections: u32,
+}
```

Comment:
In theory, we could turn `DATABASE_xxx` vars into `EN_DATABASE_xxx` so that the whole naming scheme is consistent, however
a) surely it's out of scope for this PR
b) with many partners already having their nodes set up and running, I don't really know if it's worth the hustle.

Right now, I think it's OK to ignore this topic completely until someone complains.